pkg/client/run.go

```diff
@@ -115,15 +116,24 @@ func createObject(cfg *rest.Config, obj *unstructured.Unstructured, mapper meta.
 		Namespaced: namespace != "",
 	}, namespace).Create(obj)
 
-	if err != nil {
-		return errors.Wrapf(err, "failed to create API resource %s", name)
-	}
-
-	logrus.WithFields(logrus.Fields{
+	log := logrus.WithFields(logrus.Fields{
 		"name":      name,
 		"namespace": namespace,
 		"resource":  resource,
-	}).Info("created object")
+	})
+
+	switch {
+	case err == nil:
+		log.Info("created object")
+	// Some resources (like ClusterRoleBinding and ClusterBinding) aren't
```

Comment:
`kubectl` does an update on the resources that already exist, do you think it's worth replicating that behavior?

Reply:
I don't. Either it's the same and everything's fine, or it's different and we risk breaking something in the background. I'd rather fail visibly rather than invisibly.

Reply:
though I guess it's more likely to be a stale run than a duplicate run...